api_spec.yaml

```diff
@@ -1377,7 +1378,140 @@ paths:
         
         '500':
           $ref: '#/components/responses/500'
-  
+
+  /recipe/userMetadata:
```

Comment:
No capital letters in path, so please change to `recipe/usermetadata`. 

Also, how about `/recipe/user/metadata` instead?

Reply:
Also, I assume that in case the user being queried for doesn't exist, the result will be an empty metadata object?